Examples/Tutorials/Tutorial-HelloWorld/CMakeLists.txt

```diff
@@ -1,6 +1,17 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
I think we can duplicate this for the rest of the tutorials

Reply:
done